pages/common/hub.md

```diff
@@ -11,6 +11,14 @@
 
 `hub clone {{github_username}}/{{repo_name}}`
 
+- Fork repository cloned from another user (creates a remote named after your GitHub username):
```

Comment:
`hub fork` does not ask for any extra information. You must be in a folder where the remote `origin` is the original GitHub repo (e.g. `https://github.com/tldr-pages/tldr.git`), then it will create a fork under your account and add it as a new remote named after your username (in my case, `ericbn 	git@github.com:ericbn/tldr.git`). Kind of complex to explain, but basically an automation of what you would get with the Fork button in GitHub...